hfysubs/submissions.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, unicode_literals
+
+from .reddit import make_reddit
+from beetusbot import config
+from .tasks import process_submission
+from .util import filter_post
+
+
+def handle_subscription_stream():
+    subreddit = make_reddit().subreddit(config.SUBREDDIT)
+
+    latest = config.get_latest()
+
+    for submission in subreddit.stream.submissions():
+        check = filter_post(submission)
+        if submission.id != latest and check and submission.author is not None:
```

Comment:
latest is in here from an old method.  with using the stream we shouldn't need to check this.

latest was the Hex number in the web address to define a specific post. like 6dn2wr defines [this](https://www.reddit.com/r/HFY/comments/6dn2wr/life_with_my_2d_waifu_13_aiverse/)